lsp-types/test/URIFilePathSpec.hs

```diff
@@ -249,7 +265,7 @@ genValidUnicodeChar = arbitraryUnicodeChar `suchThat` isCharacter
   where isCharacter x = x /= '\65534' && x /= '\65535'
 
 normalizedFilePathSpec :: Spec
-normalizedFilePathSpec = do
+normalizedFilePathSpec = beforeAll (setFileSystemEncoding utf8) $ do
```

Comment:
Does something actually fail if you don't do this?

Reply:
Yes, default file system encoding ignores the failure. See: https://hackage.haskell.org/package/base-4.17.0.0/docs/src/GHC.IO.Encoding.html#initFileSystemEncoding
<img width="842" alt="image" src="https://user-images.githubusercontent.com/16440269/187663305-ec633a09-db34-4937-8bf7-ddcdf3525669.png">

So in the real world, it won't throw any exception. (But I want to test the exception)